### ISSUE TITLE
Can you provide documentation for this please? Having trouble using it.

### ISSUE THREAD

**Issue by tommygaessler** [SolutiaConsulting/ng2-tweet#1]
Please let us know how to use this!

**Comment by Chewbee** [comment]
I cannot say any better 
Shall we install ng-tweet  ? 
How to declare  ? 
I a m trying but any clues welcomed ;) 
thanks to tommy gaessler for fixing here https://github.com/tommygaessler/ng2-tweet

**Comment by tommygaessler** [comment]
Yes I was able to figure it out and provide documentation! View README here: https://github.com/tommygaessler/ng2-tweet
Your welcome @Chewbee! @SolutiaConsulting please merge my pull request.

**Comment by ruucm-working** [comment]
Thanks, Tommy. It works :)

**Comment by hhmultimediallc** [comment]
@tommygaessler FANTASTIC!!!  Thank you so much!

**Comment by MickL** [comment]
Note: `import { Ng2TweetService } from 'ng2-tweet/lib/index';` -> not just `from 'ng2-tweet'`.
Also import and dependency injection within component is not needed!
